lib/metadata_json_lint.rb

```diff
@@ -154,6 +154,15 @@ def validate_requirements!(requirements)
 
       validate_puppet_ver!(puppet_req) unless puppet_req.instance_variable_get('@requirement').nil?
     end
+
+    names = requirements.map { |x| x['name'] }
+    counts = Hash.new(0)
+
+    names.each { |name| counts[name.downcase] += 1 }
+
+    counts.each { |k, v|
+      error :requirements, "Duplicate entries in the 'requirements' list with the name '#{k}'" if v > 1
+    }
```

Comment:
Lint is unhappy with you
```suggestion
    counts.each do |k, v|
      error :requirements, "Duplicate entries in the 'requirements' list with the name '#{k}'" if v > 1
    end
```